### ISSUE TITLE
Negative Accuracy for Coco_metrics

### ISSUE THREAD

**Issue by paviddavid** [tensorflow/models#6596]
Hello, 

I am using Tensorflow Obejct Detection API. A Faster RCNN trained on Kitti data and and want to make an inference on Kitti data (just to see if the code and everything works). However, I obtain an accuracy of -1.000 for the Coco_metrics, see attached image. Does anyone have an idea why this accuracy is negative and what does it mean to get a negative accuracy? I use a pretrained Kitti model (ResNet101) and Kitti validation data therefore, I expect a high accuracy for all classes. What does it mean to obtain negative values?

![image](https://user-images.githubusercontent.com/45428272/56303237-cc4bdf80-613b-11e9-82b2-8ac8e7712da9.png)

Thanks a lot in advance. 
Best Regards

**Comment by paviddavid** [comment]
@rootkitchao Thanks a lot for your fast response! Very helpful. 
But to clarify (to check if I got it in the correct way..)
The Coco dataset provides objects in different sizes and therefore, accuracy values can be calculated. Kitti only provides large objects (larger than 40 pixels https://github.com/facebookresearch/Detectron/issues/572) and therefore, no values for small and medium can be calculated, am I right? 

Another issue came up when I used the model trained on Kitti data and made an inference on Cityscapes data. I get always the same error in ```
/object_detection/utils/object_detection_evaluation.py line 260 in add_single_ground_truth_image_info
       (groundtruth_dict[standard_field.InputDataFields.groundtrutz_difficult]
AttributeError:'NoneType' object has no attribute 'size'
```

Do you have an idea why changing only the input data (everything else stays the same) leads to an error? Changing the metrics from `pascal_voc `to `coco `does not lead to this error and I get accuracy values however, I do need pascal values. 


I would be very thankful if you could help me!

Thanks a lot in advance

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
